README.md

```diff
@@ -204,13 +205,26 @@ From here, you can open up `Hyperspace.xcworkspace` and run the examples:
 
 ## Installation
 
+### Cocoapods
```

Comment:
Super minor, but let's add a blank line after this so that the formatting is consistent between the "Cocoapods" and "Carthage" sections.

Reply:
Fixed!